conductor-client/src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt

```diff
@@ -431,7 +431,7 @@ class PostgresEntityDataQueryService(
 
             //Make data visible by marking new version in ids table.
 
-            val updatedEntities = dataSourceResolver.getDataSource(DEFAULT_DATASOURCE).connection.use { idsConnection ->
+            val updatedEntities = hds.connection.use { idsConnection ->
```

Comment:
You don't need to do `hds.connection` here because you can reuse the existing `connection` object, right? That way, you avoid opening a new connection to postgres.

Reply:
Good catch! Fixed.